engine/src/rl/binaryio.py

```diff
@@ -171,6 +171,14 @@ def _set_uci_param(self, name: str, value):
                                                                  bytes(str(value), encoding="utf-8")))
         self.proc.stdin.flush()
 
+        # Log how many tablebases could be found
+        if name == f'SyzygyPath' and value != '' and value != '<empty>':
+            while True:
```

Comment:
This could in principle become a deadlock issue.
Is it possible to reduce this for only the X next lines?

Reply:
Thank you for pointing this out. I have replaced the while loop with a for loop in #132. 

The number of X next lines has to be quite big as the output from the subprocess gets buffered. For every option that we have set so far, we get at least one return statement. And because CrazyAra has a lot of UCI options that will be set (roughly 50), I think 200 is a good upper bound.